components/suggestion/command_provider/command_provider.tsx

```diff
@@ -102,21 +99,16 @@ export default class CommandProvider extends Provider {
         super();
```

Comment:
Unrelated, but shouldn't super be called with the props?

Reply:
It doesn't need to be in this case since the parent class's constructor doesn't have any parameters. Calling the parameters "props" in this case is kind of misleading since it makes it sound like a component when its not.